### DISCUSSION TITLE
Connecting to Desktop PBI model. Geting "same key" error

### DISCUSSION THREAD

**Discussion by DrYSG** [DaxStudio/DaxStudio#1223] [General]
The PBI model and report works fine. But DAX studio gies a "same key" error. Any idea how to track this down and fix?

![image](https://github.com/DaxStudio/DaxStudio/assets/2865368/f0d1b831-0b9f-4271-bff5-1d32a1369243)

**Comment by dgosbell** [comment]
This issue has already been fixed in v3.0.11 as per issue #1200
